src/fsharp/NameResolution.fs

```diff
@@ -3078,54 +3077,71 @@ let rec ResolvePartialLongIdentInModuleOrNamespace (ncenv: NameResolver) nenv is
     
     let tycons = 
         mty.TypeDefinitions
-        |> List.filter (fun tcref -> not (tcref.LogicalName.Contains(",")))
-        |> List.filter (fun tycon -> not (IsTyconUnseen ad g ncenv.amap m (modref.NestedTyconRef tycon)))
+        |> List.filter (fun tcref -> 
+            not (tcref.LogicalName.Contains(",")) &&
```

Comment:
Style: should we put the operators && || at beginning of lines? easier to scan IMHO.